testing/integration_test.go

```diff
@@ -92,7 +96,12 @@ cd cloud-profiler-nodejs
 retry git fetch origin {{if .PR}}pull/{{.PR}}/head{{else}}{{.Branch}}{{end}}:pull_branch
 git checkout pull_branch
 git reset --hard {{.Commit}}
-retry npm install --nodedir="$NODEDIR" >/dev/null
+{{if.BinaryHost}}
+retry npm install --nodedir="$NODEDIR" --fallback-to-build=false --google_cloud_profiler_binary_host_mirror={{.BinaryHost}} >/dev/null
+{{else}}
+retry npm install --nodedir="$NODEDIR" --build-from-source=profiler >/dev/null
```

Comment:
Where is this "profiler" string coming from? Mildly curious whether it should be google_cloud_profiler like in the flag name.

Reply:
Is it expected that the test still passed with the wrong string?

Reply:
Since the test is currently specifying .BinaryHost, that code wasn't exercised.

Reply:
Maybe remove that branch and code then? And add later if needed?

Reply:
Done.

Reply:
I actually realized I don't want to specify BinaryHost with the v8 canary test. Keeping this.

Reply:
Leaving this unresolved, but I will run v8-canary tests with this PR to test this code path once https://github.com/googleapis/cloud-profiler-nodejs/pull/381 is in.

Reply:
v8 canary test has been run with this PR: http://sponge.corp.google.com/34506eaf-e32f-496b-8c7f-09aa3b29d065